src/main/java/com/czertainly/core/service/impl/CertificateServiceImpl.java

```diff
@@ -1197,49 +1211,50 @@ public void removeCertificateUser(UUID userUuid) {
 
     @Override
     @ExternalAuthorization(resource = Resource.CERTIFICATE, action = ResourceAction.LIST, parentResource = Resource.RA_PROFILE, parentAction = ResourceAction.LIST)
-    public Long statisticsCertificateCount(SecurityFilter filter) {
+    public Long statisticsCertificateCount(SecurityFilter filter, boolean includeArchived) {
         setupSecurityFilter(filter);
-        return certificateRepository.countUsingSecurityFilter(filter);
+        return certificateRepository.countUsingSecurityFilter(filter, includeArchived);
```

Comment:
This should be handled by adding predicate for that field to additionalWhereClause.